apps/web/store/archive-queue.ts

```
@@ -1,103 +1,146 @@
 import { atomWithStorage, createJSONStorage } from "jotai/utils";
+import pRetry from "p-retry";
 import { jotaiStore } from "@/store";
 import { emailActionQueue } from "@/utils/queue/email-action-queue";
 import {
   archiveThreadAction,
   trashThreadAction,
   markReadThreadAction,
 } from "@/utils/actions/mail";
+import { isActionError, ServerActionResponse } from "@/utils/error";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)